lldb/include/lldb/Core/Progress.h

```diff
@@ -90,10 +90,11 @@ class Progress {
   /// @param [in] amount The amount to increment m_completed by.
   ///
   /// @param [in] an optional message associated with this update.
-  void Increment(uint64_t amount = 1, std::string update = {});
+  void Increment(uint64_t amount = 1,
+                 std::optional<std::string> updated_detail = {});
```

Comment:
`updated_detail` is still kept as an empty string by default, though I'm unsure if the empty string or `std::nullopt` is preferred here